--- liborcus-0.5.1/src/liborcus/Makefile.am.omv~	2014-03-22 18:08:54.343398825 +0100
+++ liborcus-0.5.1/src/liborcus/Makefile.am	2014-03-22 18:09:37.110397074 +0100
@@ -175,7 +175,7 @@ liborcus_@ORCUS_API_VERSION@_la_CXXFLAGS
 	$(BOOST_SYSTEM_CFLAGS) $(ZLIB_CFLAGS)
 liborcus_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
 liborcus_@ORCUS_API_VERSION@_la_LIBADD = \
-	 $(BOOST_SYSTEM_LIBS) $(BOOST_SYSTEM_LIB) $(ZLIB_LIBS)
+	 $(BOOST_SYSTEM_LIBS) $(BOOST_SYSTEM_LIB) $(ZLIB_LIBS) ../parser/liborcus-parser-@ORCUS_API_VERSION@.la
 
 # liborcus-test-xml-map-tree
 
--- liborcus-0.5.1/src/mso/Makefile.am.omv~	2014-03-22 18:07:11.487403036 +0100
+++ liborcus-0.5.1/src/mso/Makefile.am	2014-03-22 18:07:44.702401676 +0100
@@ -7,4 +7,4 @@ liborcus_mso_@ORCUS_API_VERSION@_la_SOUR
 
 liborcus_mso_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
 liborcus_mso_@ORCUS_API_VERSION@_la_LIBADD = \
-	../parser/liborcus-parser-@ORCUS_API_VERSION@.la
+	../parser/liborcus-parser-@ORCUS_API_VERSION@.la -lboost_system
--- liborcus-0.5.1/src/parser/Makefile.am.omv~	2014-03-22 18:08:06.914400767 +0100
+++ liborcus-0.5.1/src/parser/Makefile.am	2014-03-22 18:08:27.614399919 +0100
@@ -20,7 +20,7 @@ liborcus_parser_@ORCUS_API_VERSION@_la_S
 
 liborcus_parser_@ORCUS_API_VERSION@_la_LDFLAGS = -no-undefined
 liborcus_parser_@ORCUS_API_VERSION@_la_LIBADD = \
-	 $(ZLIB_LIBS)
+	 $(ZLIB_LIBS) -lboost_system
 
 EXTRA_PROGRAMS = \
 	parser-test-string-pool \
