diff -up liborcus-0.16.1/src/liborcus/orcus_xlsx.cpp.omv~ liborcus-0.16.1/src/liborcus/orcus_xlsx.cpp
--- liborcus-0.16.1/src/liborcus/orcus_xlsx.cpp.omv~	2021-08-17 23:48:57.921581229 +0200
+++ liborcus-0.16.1/src/liborcus/orcus_xlsx.cpp	2021-08-17 23:49:42.315671821 +0200
@@ -35,6 +35,7 @@
 #include <string>
 #include <cstring>
 #include <sstream>
+#include <limits>
 
 using namespace std;
 
diff -up liborcus-0.16.1/src/liborcus/xls_xml_context.cpp.omv~ liborcus-0.16.1/src/liborcus/xls_xml_context.cpp
--- liborcus-0.16.1/src/liborcus/xls_xml_context.cpp.omv~	2021-08-17 23:51:56.257945144 +0200
+++ liborcus-0.16.1/src/liborcus/xls_xml_context.cpp	2021-08-17 23:52:01.889956637 +0200
@@ -16,6 +16,7 @@
 #include <mdds/sorted_string_map.hpp>
 
 #include <iostream>
+#include <limits>
 
 using namespace std;
 namespace ss = orcus::spreadsheet;
diff -up liborcus-0.16.1/src/liborcus/xlsx_revision_context.cpp.omv~ liborcus-0.16.1/src/liborcus/xlsx_revision_context.cpp
--- liborcus-0.16.1/src/liborcus/xlsx_revision_context.cpp.omv~	2021-08-17 23:50:56.269822733 +0200
+++ liborcus-0.16.1/src/liborcus/xlsx_revision_context.cpp	2021-08-17 23:51:03.131836736 +0200
@@ -16,6 +16,7 @@
 #include "orcus/global.hpp"
 
 #include <iostream>
+#include <limits>
 
 using namespace std;
 
